### ISSUE TITLE
Moderate fatigue

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5925]
**Preferred term label:** Moderate fatigue

**Synonyms**

**Definition (free text, please give PubMed ID)** A subjective feeling of moderate tiredness characterized by a lack of energy and motivation., PMID:27428862

**Parent term (use hpo.jax.org/app)** Fatigue HP:0012378

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Please use modifiers to express mild/moderate/severe -- https://hpo.jax.org/app/browse/term/HP:0012824
